channeld/channeld_wire.csv

```diff
@@ -181,6 +201,72 @@ msgdata,channeld_got_revoke,penalty_tx,?bitcoin_tx,
 msgtype,channeld_got_revoke_reply,1122
 
 #include <wally_bip32.h>
+
+# master->channeld: hello, I'd like to start a channel splice open
+msgtype,channeld_splice_init,7204
+msgdata,channeld_splice_init,psbt,wally_psbt,
+msgdata,channeld_splice_init,funding_amount,amount_sat,
+msgdata,channeld_splice_init,feerate_per_kw,u32,
+msgdata,channeld_splice_init,force_feerate,bool,
+
+# channeld->master: hello, I started a channel splice open
+msgtype,channeld_splice_confirmed_init,7205
+msgdata,channeld_splice_confirmed_init,psbt,wally_psbt,
+
+# master->channeld: Update an active splice
+msgtype,channeld_splice_update,7206
+msgdata,channeld_splice_update,psbt,wally_psbt,
+
+# channeld->master: Splice update complete
+msgtype,channeld_splice_confirmed_update,7207
+msgdata,channeld_splice_confirmed_update,psbt,wally_psbt,
+msgdata,channeld_splice_confirmed_update,commitments_secured,bool,
+
+# channeld->master: Lookup a transaction
+msgtype,channeld_splice_lookup_tx,7208
+msgdata,channeld_splice_lookup_tx,txid,bitcoin_txid,
+
+# master->channeld: Retrieved transaction
+msgtype,channeld_splice_lookup_tx_result,7209
+msgdata,channeld_splice_lookup_tx_result,tx,bitcoin_tx,
+
+# master->channeld: User has signed psbt and it's ready to complete
+msgtype,channeld_splice_signed,7212
+msgdata,channeld_splice_signed,psbt,wally_psbt,
+msgdata,channeld_splice_signed,force_sign_first,bool,
+
+# channeld->master: Signed psbt is completed
+msgtype,channeld_splice_confirmed_signed,7213
+msgdata,channeld_splice_confirmed_signed,tx,bitcoin_tx,
+msgdata,channeld_splice_confirmed_signed,output_index,u32,
+
+# channeld->master: A feerate error has occured
+msgtype,channeld_splice_feerate_error,7215
+msgdata,channeld_splice_feerate_error,fee,amount_sat,
+msgdata,channeld_splice_feerate_error,too_high,bool,
+
+# channeld->master: Add an inflight to the DB
+msgtype,channeld_add_inflight,7216
+msgdata,channeld_add_inflight,tx_id,bitcoin_txid,
+msgdata,channeld_add_inflight,tx_outnum,u32,
+msgdata,channeld_add_inflight,feerate,u32,
+msgdata,channeld_add_inflight,satoshis,amount_sat,
+msgdata,channeld_add_inflight,our_funding_satoshis,amount_sat,
+msgdata,channeld_add_inflight,psbt,wally_psbt,
```

Comment:
I thiiiink this comment is related to the convo about storing inflights with/without commit+HTLC sigs, and since we moved to storing them without we can mark this one resolved 👀